README.md

```diff
@@ -25,14 +25,14 @@ echo 'MNEMONIC="bike north stone..."' > .env
   See the [supported networks section](#supported-networks) for valid `NETWORK` values.
 
 ```sh
-NETWORK=polygon-testnet PROXY=0x009E9B1eec955E9Fe7FE64f80aE868e661cb4729 yarn deploy
+NETWORK=ethereum-sepolia-testnet PROXY=0x51097c0ed085958897fb2B062961E54B1D4d5771 yarn deploy
```

Comment:
> Where did you get this address from?
While I was going thru the README to confirm that everything in there still applies after phase 2.1 launch, I deployed a proxy contract on the market UI for BTC/USD and that was the resulting address.

> It will be difficult to update this README properly until https://github.com/api3dao/data-feeds/issues/111 is done.
I wasn't aware of that change. Thanks for pointing it out.  I guess the scripts for deploying proxies will become more relevant in this README and should even be moved to before using the script to read the data feed value

Reply:
The Market sends the user to this repo so I think having anything in the README that the Market doesn't support is not a good idea (so the proxy deployment command in the README should do the same thing as the Market, deploy a DapiProxyWithOev where the beneficiary is the manager multisig so that the user gets the same proxy address and isn't confused). DataFeedReaderExample, specifying beneficiary, etc. can be documented in an advanced section/a separate md file.

Reply:
https://github.com/api3dao/data-feed-reader-example/pull/61/commits/4ed64abb6ead628c9a4fd699af975a6001869472